Owncloud iOs Client/InstantUpload/InstantUpload.m

```diff
@@ -0,0 +1,215 @@
+//
+//  InstantUpload.m
+//  Owncloud iOs Client
```

Comment:
Same here:
You have to use the header that we use on the other files. Check any other class and copy it.
Of course you can put your name as "created" 😉